rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +19,25 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise serializers.ValidationError(msg)
+                    raise serializers.ValidationError(
+                        ValidationErrorMessage(
+                            msg,
+                            code='authorization')
```

Comment:
Well it could be the same if I'd move the logic that actually constructs the `ValidationErrorMessage` within `ValidationError.__init__()`.

So, I could change the use of `ValidationError` without breaking anything indeed. And that would also enforce that `ValidationError.detail` will always be a `ValidationErrorMessage`.